litmus-portal/graphql-server/graph/analytics.graphqls

```diff
@@ -108,15 +121,49 @@ input promQueryInput {
     minstep: Int!
 }
 
-type promResponse {
+type metricsPromResponse {
     queryid: String!
-    legends: [[String]]
-    tsvs: [[timeStampValue]]
+    legends: [String]
+    tsvs: [[metricsTimeStampValue]]
+}
+
+type metricsTimeStampValue{
+    date: Float
+    value:  Float
+}
+
+type annotationsPromResponse {
+    queryid: String!
+    legends: [String]
+    tsvs: [[annotationsTimeStampValue]]
+}
+
+type annotationsTimeStampValue{
+    date: Float
+    value:  Int
```

Comment:
I think we shouldn't use Int for timestamp.

Reply:
yes we have updated to float but we cannot use string due to extra parsing overhead, flat of 64 bits may be long enough for holding the timestamp values.